src/content/docs/licenses/license-information/usage-plans/product-based-pricing-usage-new-relic-platform-pricing-usage-plan.mdx

```diff
@@ -302,7 +302,7 @@ The following Eligible Services with New Relic are products that are referenced
 
     <tr>
       <td>
-        New Relic AI Incident Intelligence Annual
+        New Relic applied intelligence Incident Intelligence Annual
```

Comment:
I'm wondering if we should capitalize applied intelligence here and in the next one?